data_layer/celestia/celestia.go

```diff
@@ -191,6 +217,13 @@ func (c *Celestia) GetNetworkName() string {
 }
 
 func (c *Celestia) GetSequencerDAConfig() string {
+	lcEndpoint := c.GetLightNodeEndpoint()
 	return fmt.Sprintf(`{"base_url": "%s", "timeout": 60000000000, "gas_prices":0.1, "gas_adjustment": 1.3, "namespace_id":"000000000000ffff"}`,
```

Comment:
it's better to randomize the `namespace_id` to prevent all rollapps writing to the same one. though possible not ideal as redundant batches may get picked up if multiple rollapps wrote to the same DA block.